test/spec_helper.rb

```diff
@@ -26,7 +26,9 @@
 
 if ENV["COVERAGE"]
   require "simplecov"
-  SimpleCov.start
+  SimpleCov.start do
+    add_filter "/test/"
```

Comment:
For the record, this is the cause of the coverage drop which results in the "build failed" X mark. It works as intended since we do want to stop counting tests as covered code.